OpenSim/Simulation/Model/Force.cpp

```diff
@@ -56,7 +56,29 @@ void Force::setNull()
 // Define properties.
 void Force::constructProperties()
 {
-    constructProperty_isDisabled(false);
+    constructProperty_appliesForce(true);
+}
+
+void
+Force::updateFromXMLNode(SimTK::Xml::Element& node, int versionNumber) {
+    if(versionNumber < XMLDocument::getLatestVersion()) {
+        if(versionNumber < 30508) {
+            // Rename property 'isDisabled' to 'appliesForce' and
+            // negate the contained value.
+            std::string oldName{"isDisabled"};
+            std::string newName{"appliesForce"};
+            if(node.hasElement(oldName)) {
+                auto elem = node.getRequiredElement(oldName);
+                elem.setElementTag(newName);
+                if(elem.getValue().find("true") != std::string::npos)
+                    elem.setValue("false");
+                else
+                    elem.setValue("true");
```

Comment:
This logic looks pretty good to me. A question: If someone has `<isDisabled></isDisabled>` in their XML file, should we interpret this as `appliesForce == true`, as is currently done? What would happen before this PR if the XML file has `<isDisabled></isDisabled>`?  

For reference, this is the code that Simbody uses to parse bool properties (I think):

```cpp
bool String::tryConvertToBool(bool& out) const {                                
    const String adjusted = cleanUp(*this);                                     
    if (adjusted=="true")  {out=true;  return true;}                            
    if (adjusted=="false") {out=false; return true;}                            
    std::istringstream sstream(adjusted);                                       
    sstream >> out;                                                             
    return !sstream.fail();                                                     
}   
```

https://github.com/simbody/simbody/blob/master/SimTKcommon/src/String.cpp#L81

Thus, a more robust way to perform this conversion might be:

```cpp
elem.setValue(SimTK::String(!SimTK::String::tryConvertToBool(elem.getValue())));
```